### PR TITLE
Add Pattern35.py to the repository

### LINKED ISSUE #46: Add star pattern in Python, Pattern35
# Description
## Task
For the given input n the number of rows printed should be n for ex:

### PR DESCRIPTION
**Pull request by MirrorCraze**
Add the code for Pattern 35 in Python

This commit resolved #46

### CODE REVIEW

File: pattern35.py
Comment (ksharma20) [inline pattern35.py:1]: File is Empty ?

Comment (MirrorCraze) [reply pattern35.py:1]: I pushed to the wrong branch before. The file is added to the right branch now!

**Comment by MirrorCraze** [comment]
@Vishal21121 Rename is done!

**Review by Vishal21121 — APPROVED** [review]
@MirrorCraze Code works fine 👍

**Comment by Vishal21121** [comment]
@MirrorCraze you can star this repo if you liked it

**Pull request merged**